doc/040_backup.rst

```diff
@@ -187,6 +187,23 @@ On **Windows**, a file is considered unchanged when its path and modification
 time match, and only ``--force`` has any effect. The other options are
 recognized but ignored.
 
+Dry Runs
+********
+
+You can perform a backup in dry run mode to see what would happen without
+modifying the repo.
+
+-  ``--dry-run``/``-n`` do not write anything, just print what would be done
+
+Combined with ``--verbose``, you can see a list of changes:
```

Comment:
why does `--dry-run` have to be combined with `--verbose`?

Reply:
It does not have to be combined. `backup --dry-run` gives the same output as `backup` without dry-run (except it replaces the sentence "Added to the repo" by "Would be added to the repo"). But if you want to see the changes by details, you have to run it with increased verbosity.

If you think that this part of the docu is unclear: Do have a suggestion how to describe it?

Reply:
This may just be a little linguistic confusion around "do not **write** anything"?

A few alt takes:
- `report what would be done without modifying the repository`
- `report what would be done, but do not modify the repository`
- `report what would be done without doing it`
- `report what would be done without doing anything`
- `report what would be done, but do nothing`
- `report what would be done, but make no changes`

Reply:
`report what would be done, without modifying the repository` :-)

Reply:
is changed.